CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+We welcome pull requests from everyone. By participating in this project, you
+agree to abide by our simple code of conduct - be nice to people.
+
+Fork, then clone the repository:
+
+    git clone git@github.com:your-username/development-environment-base.git
```

Comment:
'development-environment-base' is intentional, it is what the repo should be renamed to, to reduce confusion with the development-environment repo